third_party/blink/renderer/core/page/page.cc

```diff
@@ -540,6 +542,11 @@ void Page::SetVisibilityState(
     return;
   lifecycle_state_->visibility = visibility_state;
 
+  // ideally we could use an observer for this (and we might eventually find that we can),
+  // but in the interest of always getting paints, we need do this even if
+  // is_initial_state == true.
+  recordreplay::NotePageVisibilityStateChanged(this);
```

Comment:
I have never used the `Note` verbiage:
* I have always used `Notify`.
* Chromium also uses `Notify`.

I am biased to prefer `Notify` over `Note`, unless you have any objections against that?

Reply:
no strong opinion here at all.